packages/turf-dissolve/index.js

```diff
@@ -106,8 +106,10 @@ module.exports = function (featureCollection, propertyName) {
             }
             var matchFeature = featureCollection.features[matchFeaturePosition];
 
-            if (matchFeature.properties[propertyName] !== polygon.properties[propertyName]) {
-                continue;
+            if (propertyName !== null) {
```

Comment:
Should be `undefined` instead of `null.
An empty param will result as `undefined`.

**Question**: Did this get picked up in your tests?